monkvision/monkjs
# Feature/fe 250/camera simple view layout

ALL CONTRIBUTIONS WILL BE REVIEWED BY AT LEAST ONE OF OUR FRONTEND TEAM MEMBERS.







## Technical description

Added new ratio calculation logic using **euclidean GCD**.

## Related to



This PR is related to [Camera Simple View Layout](https://monkvision.atlassian.net/browse/FE-250)

## About Tests


Tested on the following devices

- iPhone 8, iPhone 8 plus
- pixel 2 XL
- Web chrome browser


Steps to reproduce

1. Land on damages screen
2. Open the add a damage form (press a vehicle part)
3. Add a picture (camera should be full screen




## Screenshots (optional):

No screenshots.
*This Pull Request template has been written and generated by Monk JS repository.*
